driven/firebase/adapter.go

```diff
@@ -27,41 +29,67 @@ import (
 
 // Adapter entity
 type Adapter struct {
-	firebase *firebase.App
+	firebaseConfs []model.FirebaseConf
+
+	//key is org-id_app-id construction
+	firebaseClients map[string]firebase.App
 }
 
 // NewFirebaseAdapter instance a new Firebase adapter
-func NewFirebaseAdapter(authFile string, projectID string) *Adapter {
-	conf, err := google.JWTConfigFromJSON([]byte(authFile),
+func NewFirebaseAdapter(firebaseConfs []model.FirebaseConf) *Adapter {
+	return &Adapter{firebaseConfs: firebaseConfs, firebaseClients: make(map[string]firebase.App)}
+}
+
+// Start starts the firebase adapter
+func (fa *Adapter) Start() error {
```

Comment:
What if we want to add, update, or remove a Firebase config for a new or existing app or org while the service is running? Ideally I think we would want a "system" API to allow these operations to occur. Once these APIs are called it would be great if the changes were reflected right away without needing to restart the service.